package.json

```diff
@@ -31,6 +31,7 @@
     "build:noclean": "ts-node -T tools/build --verbose",
     "docs": "npm run build && rimraf docs/_build && cd docs && sphinx-build . _build -a",
     "docs:dev": "npm run build:dev && rimraf docs/_build/* && cd docs && sphinx-build . _build -a",
+    "docs:watch": "ts-node -T tools/build --verbose --dirty --watch --docs",
```

Comment:
? Is this intended to mimic `mkdocs serve`?

I don't really use these commands, but I'm trying to learn to.

Reply:
This doesn't actually start an http server for hosting the docs --- you still have to do that separately if you need it.  But it does auto-rebuild the docs if you make any changes to any of the sources.  That way you just have to refresh the web browser to see the changes.